src-built-in/components/toolbar/stores/toolbarStore.js

```diff
@@ -161,18 +170,15 @@ class _ToolbarStore {
 	 */
 	addListeners(done, self) {
 		// menus change - menus come from config
-		FSBL.Clients.DistributedStoreClient.getStore({ store: "Finsemble-Configuration-Store", global: true }, function (err, configStore) {
-			if (configStore) {
-				configStore.addListener({ field: "finsemble.menus" }, function (err, data) {
-					self.Store.setValue({
-						field: "menus",
-						value: data.value
-					});
-					self.getSectionsFromMenus(data.value);
-				});
-			}
-			done();
+		FSBL.Clients.ConfigClient.addListener({ field: "finsemble.menus" }, function (err, data) {
+			if (err) { FSBL.Clients.Logger.error(`DistributedStoreClient.getStore -> configStore.addListener failed for Finsemble-Configuration-Store, error:`, err); }
```

Comment:
Looking at https://github.com/ChartIQ/finsemble/blob/master/src/clients/configClient.ts#L230
It seems that the chance for an error to be a non null value is 0%. We don't really need all these if(err) checks and log messages.

Reply:
This PR found several errors that we were never anticipated. I asked Zona to put these here in case we change the config client in the future.